### PR TITLE
[dom-gpu] Add Amber 20 with CUDA

### PR DESCRIPTION
**Pull request by victorusu**

### CODE REVIEW

**Comment by victorusu** [comment]
@jenkins-cscs retest this please

**Comment by victorusu** [comment]
retest this please

**Comment by victorusu** [comment]
I cannot understand why Jenkins doesn't build this software version.
I am able to install the software with my user.
```------------------------------------ /apps/daint/UES/sandbox/hvictor/7.1/easybuild/modules/all ------------------------------------
Amber/20-3-0-CrayIntel-19.10-cuda-10.1
```

**Comment by lucamar** [comment]
@victorusu Did you build it on the compute nodes of Piz Daint with your username?

File: jenkins-builds/7.0.UP02-20.06-gpu-dom [hunk 1]
```diff
@@ -1,3 +1,4 @@
+ Amber-20-3-0-CrayIntel-20.06-cuda-10.2.eb           --set-default-module
```
Comment (lucamar) [inline jenkins-builds/7.0.UP02-20.06-gpu-dom:1]:
```suggestion
 Amber-20-3-0-CrayIntel-20.06-cuda.eb           --set-default-module
```

File: easybuild/easyconfigs/a/Amber/Amber-20-3-0-CrayIntel-20.06-cuda-10.2.eb [hunk 2]
```diff
@@ -0,0 +1,97 @@
+# @author: Victor Holanda
+
+easyblock = 'MakeCp'
+
+name = 'Amber'
+local_patchlevels = (3, 0) # (AmberTools, Amber)
+version = '20-%s-%s' % (local_patchlevels[0], local_patchlevels[1])
+local_cudaversion = '10.2'
+versionsuffix = '-cuda-%s' % local_cudaversion
```
Comment (lucamar) [inline easybuild/easyconfigs/a/Amber/Amber-20-3-0-CrayIntel-20.06-cuda-10.2.eb:9]:
```suggestion
versionsuffix = '-cuda'
```

**Pull request merged**